staging/src/k8s.io/api/go.mod

```diff
@@ -5,12 +5,13 @@ module k8s.io/api
 go 1.12
 
 require (
-	github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
+	github.com/gogo/protobuf v1.0.0
 	github.com/stretchr/testify v1.3.0
 	k8s.io/apimachinery v0.0.0
 )
 
 replace (
+	github.com/gogo/protobuf => github.com/gogo/protobuf v0.0.0-20171007142547-342cbe0a0415
```

Comment:
Hmmm… this is unexpected. This means we did not actually bump the dependency version. Did you use hack/pin-dependency.sh to update it to v1.0.0?

Reply:
Whoops, sorry, I was still using gomod commands manually. I'll try that again.